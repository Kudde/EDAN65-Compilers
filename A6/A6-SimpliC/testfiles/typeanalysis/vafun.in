int a(int o) {
  return 3;
}

int fun () {
  int b = 1;
  int c = 2;
  a = b + 3;
  c();
}

int b () {
  return 54;
}
