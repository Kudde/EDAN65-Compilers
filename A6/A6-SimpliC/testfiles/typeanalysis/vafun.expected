Error at line 8: symbol 'a' is a function!
Error at line 9: symbol 'c' is not a function!
